src/Symfony/Component/String/Slugger/AsciiSlugger.php

```diff
@@ -88,6 +90,18 @@ public function getLocale(): string
         return $this->defaultLocale;
     }
 
+    public function withEmojiSupport(bool $enableEmoji = true): static
```

Comment:
just `withEmoji(bool $emoji = true)`?
also, what about making it `withEmoji(bool|string $emoji = true)` to allow passing the emoji "locale" when the slugger's locale doesn't exactly match the emoji's? (eg "slack")